app/views/users/_profile.html.erb

```diff
@@ -103,7 +154,13 @@
                               "profile[#{field.attribute_name}]",
                               value: profile.public_send(field.attribute_name),
                               placeholder: field["placeholder_text"],
+                              aria: { describedby: "#{field.attribute_name}-description" },
+                              data: { character_span_id: "#{field.attribute_name}-characters" },
                               class: "crayons-textfield") %>
+            <div id="<%= field.attribute_name %>-description" class="self-end fs-s">
+              <span class="screen-reader-only">Available characters used: </span>
+              <span id="<%= field.attribute_name %>-characters"></span>/<%= field["input_type"] == "text_field" ? "100" : "200" %>
```

Comment:
I coded the characters denominator thus, given that custom ProfileField types are `text_field` or `text_area`. See the [form](https://github.com/forem/forem/blob/main/app/views/admin/profile_fields/_profile_field_form.html.erb) and run `ProfileField.input_types.keys` to confirm.

Reply:
I think I would feel a bit better if we moved this to a helper or decorator. The less logic we have in templates the better I think.